cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        f['resolutions']['1000000']['bins']['weight_inverted'] = 1/f['resolutions']['1000000']['bins']['weight'][:]
</pre><p>should the reading mode be described?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
move to line above for readability, open hdf5 file in r+ mode which allows read/write but the file must exist